src/frontend.rs

```diff
@@ -592,6 +593,7 @@ impl AsyncComponent for App {
 
                 T.about_system_information(config_directory, data_directory)
                     .as_str()
+                    .to_string()
             })
```

Comment:
This should work instead without heap allocations:

```suggestion
            }.as_str())
```

It is only about the scope after which variable is dropped.